### DISCUSSION TITLE
Why is CHANGES.md used in cd.yml and not CHANGELOG.md?

### DISCUSSION THREAD

**Discussion by tessus** [orhun/git-cliff#738] [Q&A] [answered]
Why is `CHANGES.md` used in [cd.yml](https://github.com/orhun/git-cliff/blob/main/.github/workflows/cd.yml#L26), even though `CHANGELOG.md` is in the repo and no `CHANGES.md` can be found anywhere?

**Comment by orhun** [comment, marked as answer]
It's an arbitrary file. I initially copied it from `git-cliff-action` repo and at some point used it for the release notes but now I simply use `steps.git-cliff.outputs.content` so I'm guessing [it can be removed](https://github.com/orhun/git-cliff/commit/9f39bcb8bf87ede149c11be0eb913583ded191c5).
